github-connector/internal/events/sender_test.go

```diff
@@ -0,0 +1,168 @@
+package events_test
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/apperrors"
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/events"
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/events/mocks"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type toJSON struct {
+	TestJSON string `json:TestJSON"`
+}
+
+type ClientMock struct {
+}
+
+func (c *ClientMock) Do(req *http.Request) (*http.Response, error) {
+	return &http.Response{}, nil
+}
+
+func TestSendToKyma(t *testing.T) {
```

Comment:
In general I identified the following test cases to cover:
- everything was ok, call to event service sent
- validation failed (Validator mock returns error)
- failed to send request (client.Do returned an error) : e.g. address is incorrect
- request has been sent but event wasn't delivered:
  - server returned 404
  - server returned 500

Actually the cases with returning 404/500 error could be handled as a one case.